content/review/document-level-permissions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Document level Permissions"
+metaTitle: "sensenet - Document level Permissions"
+metaDescription: "sensenet Document level Permissions"
+---
+
+sensenet offers you the ability of setting permissions even on document level in addition to the [role-based permission](/concepts/role-based-permissions) logic.
+# Advantage
+Using content (or document) level permissions, you can set who has access to an individual content and allow/restrict actions one by one.
+
+# Built-in and custom permissions
+
+There is a set of basic built-in permissions, such as See, Open, Save, Delete etc., and it is even possible to add custom ones as well.
+•	See: you only have the information that the content exists (content metadata cannot be accessed)
+•	Open: you have access to content metadata
+•	Save: you can make changes in the content metadata
+•	Other permissions for removing or approving a content or managing lists and workspaces.
+
+# Permission inheritance
+
+One of the advantages of the tree based hierarchical content structure in sensenet, is that it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents.
+
+# How to use
+
+It is possible to manage content (document) level permission settings through the admin surface, or with User Management and Permission APIs.
+
+# Pro tip
+
+Useful when you want to give access to a document only for specific users (and not by role).
```

Comment:
The last two paragraphs to short to be a separate paragraph with title and stuff. Paste them somewhere else above.